### ISSUE TITLE
Cannot copy CSI data | sudo log_to_file csi.dat not working

### ISSUE THREAD

**Issue by azizshahics** [dhalperi/linux-80211n-csitool-supplementary#167]
[Hello,](url
![Uploading FullSizeRender.jpg…]%28%29)
           Can you please help me on how to copy CSI measured data into a csi.dat file?
I would really appreciate if you help me out

**Comment by asifhanif1122** [comment]
Did you successfully install modified firmware?

**Comment by azizshahics** [comment]
Yes I succesfully modified.can you please give me your what'app or skype id ? i need to have word with you.

**Comment by azizshahics** [comment]
yes.i modified.can you please give me your whatsapp or skype? i need to have word with you. 

```
On Wednesday, April 27, 2016 2:54 AM, asifhanif1122 <notifications@github.com> wrote:
```

 Did you successfully install modified firmware?—

**Comment by asifhanif1122** [comment]
You can contact me on this email address:  asif.hanif@outlook.com
Sorrowfully , I don't have Whatsapp.
